libs/src/utils/network.js

```diff
@@ -18,7 +18,7 @@ async function timeRequest (request) {
     response = await axios.get(request.url)
   } catch (e) {
     console.debug(`Error with request for ${request.url}: ${e}`)
-    return null
+    return { request, response: null, millis: null }
```

Comment:
`timeRequests` fails in this case since it does timings.sort((a, b) => a.millis - b.millis)
need to probably add a check for null there, or just return something stupid like `Number.MAX_SAFE_INTEGER`

Reply:
we actually aren't using this anywhere anymore, but i'll add a `filter` to filter out failed health check responses

Reply:
if we aren't let's just delete the method!